SCons/Tool/gs.xml

```diff
@@ -47,7 +47,7 @@ environment.
 <cvar name="GS">
 <summary>
 <para>
-The Ghostscript program used, e.g. to convert PostScript to PDF files.
+The name of the Ghostscript program used, e.g. to convert PostScript to PDF files.
```

Comment:
This could also be the full path. Might be best to revert this change as it's a bit more accurate.